### PR TITLE
Add instructions for booting terrascan demo

### PR DESCRIPTION
**Pull request by kklin**
@cesar-rodriguez we've put together the first version of the GitHub badge!

Screen Shot 2020-09-13 at 3 27 00 PM

As far as I can tell, the booted version of Terrascan works. I was able to curl v1/terraform/v12/aws/local/file/scan.

What do you think?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -62,6 +63,17 @@ $ make build
...
+If you want to play around with Terrascan without running it locally, you can
+[boot a personal demo copy
+](https://kelda.io/preview-env/?repo=https://github.com/accurics/terrascan.git&composeFiles=deploy/docker-compose.yml&service=terrascan&port=9010)
+from your browser without downloading or setting up anything.
+
+Clicking the
+[link](https://kelda.io/preview-env/?repo=https://github.com/accurics/terrascan.git&composeFiles=deploy/docker-compose.yml&service=terrascan&port=9010)
+boots this repo in the Blimp cloud, and creates a public URL for you to access
+it.
+
```
Comment (cesar-rodriguez) [inline README.md:76]: Would you mind adding the curl example of how to interact with the sandbox once it's up?

**Review by cesar-rodriguez — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (kklin) [reply README.md:118]: Done! The example terraform file I used didn't have any violations, but if you have a different file that you think would be a better demo I can update it.

**Comment by kklin** [comment]
Hi @cesar-rodriguez, did you see the latest version of the docs?

**Review by cesar-rodriguez — APPROVED** [review]

**Review by cesar-rodriguez — APPROVED** [review]
Works great. Thanks!

**Comment by kklin** [comment]
Thanks! Let me know if you have any issues

**Pull request merged**